dxworkflow.json

```diff
@@ -182,6 +157,74 @@
           }
         }
       }
+    },
+    {
+      "id": "stage-eggd_vep_cgppindel",
+      "executable": "app-eggd_vep/1.3.0",
+      "folder": "eggd_vep",
+      "input": {
+        "vcf": {
+          "$dnanexus_link": {
+            "stage": "stage-cgppindel",
+            "outputField": "output_vcf_with_vaf"
+          }
+        }
+      }
+    },
+    {
+      "id": "stage-eggd_vep_mutect2",
+      "executable": "app-eggd_vep/1.3.0",
+      "folder": "eggd_vep",
+      "input": {
+        "vcf": {
+          "$dnanexus_link": {
+            "stage": "stage-sentieon_tnbam",
+            "outputField": "variants_vcf"
+          }
+        }
+      }
+    },
+    {
+      "id": "stage-eggd_vcf_rescue",
+      "executable": "app-eggd_vcf_rescue/1.2.0",
+      "folder": "eggd_vcf_rescue",
+      "input": {
+        "filtered_vcf": {
+          "$dnanexus_link": {
+            "stage": "stage-eggd_vep_mutect2",
+            "outputField": "annotated_vcf"
+          }
+        },
+        "unfiltered_vcf": {
+          "$dnanexus_link": {
+            "stage": "stage-eggd_vep_mutect2",
+            "outputField": "annotated_vcf"
+          }
+        }
+      }
+    },
```

Comment:
The same VCF is provided for the both filtereted and unfiltered inputs and this is expected. This is because each of the VCF will be treated slightly different, see more info here: https://cuhbioinformatics.atlassian.net/wiki/spaces/DV/pages/3138945120/eggd_vcf_rescue+v1.2.0+Testing#%F0%9F%A6%8B-Summary